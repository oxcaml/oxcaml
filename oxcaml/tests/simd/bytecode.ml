external int_array_get_int64x2 : int array -> int -> int64x2
  = "%caml_int_array_get128"

let () = int_array_get_int64x2 [| 1; 2 |] 0 |> ignore
