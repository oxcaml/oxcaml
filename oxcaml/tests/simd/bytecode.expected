Fatal error: SIMD is not supported in bytecode mode.
Exit status: 134
